webapp/components/admin_console/login_settings.jsx

```diff
@@ -74,6 +139,103 @@ export default class LoginSettings extends AdminSettings {
 
         return (
             <SettingsGroup>
+                <TextSetting
+                    id='passwordMinimumLength'
+                    label={
+                        <FormattedMessage
+                            id='admin.password.minimumLength'
+                            defaultMessage='Minimum Password Length:'
+                        />
+                    }
+                    helpText={
+                        <FormattedMessage
+                            id='admin.password.minimumLengthDescription'
+                            defaultMessage='Minimum number of characters required for a valid password.'
+                        />
+                    }
+                    value={this.state.passwordMinimumLength}
+                    onChange={this.handleChange}
+                />
+                <Setting
```

Comment:
`handleChange` is actually part of the [`AdminSettings`](https://github.com/mattermost/platform/blob/master/webapp/components/admin_console/admin_settings.jsx#L34) base class that all the settings pages use. It doesn't work with the checkboxes because they use a different set of parameters than `AdminSettings.handleChange`. You can see how it is called in [`TextSetting.onChange`](https://github.com/mattermost/platform/blob/master/webapp/components/admin_console/text_setting.jsx#L40)